code/modules/photography/camera/camera_image_capturing.dm

```diff
@@ -69,11 +73,13 @@
 		var/xo = (A.x - center.x) * world.icon_size + A.pixel_x + xcomp
 		var/yo = (A.y - center.y) * world.icon_size + A.pixel_y + ycomp
 		if(ismovableatom(A))
-			var/atom/movable/AM = A
-			xo += AM.step_x
-			yo += AM.step_y
+			var/obj/effect/appearance_clone/clone = A

```

Comment:
This is in a loop, in case you were wondering why the cast doesn't work.

Reply:
yep, they're not pretty